crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -142,30 +149,27 @@ pub async fn import(
         import.salt,
     );
 
-    write_account_to_accounts_file(account, accounts_file, chain_id, account_json.clone())?;
+    write_account_to_accounts_file(&account_name, accounts_file, chain_id, account_json.clone())?;
 
     if import.add_profile.is_some() {
         let config = CastConfig {
             url: import.rpc.url.clone().unwrap_or_default(),
-            account: account.into(),
+            account: account_name.clone(),
             accounts_file: accounts_file.into(),
             ..Default::default()
         };
         add_created_profile_to_configuration(&import.add_profile, &config, &None)?;
     }
 
     Ok(AccountImportResponse {
-        add_profile: if import.add_profile.is_some() {
-            format!(
-                "Profile {} successfully added to snfoundry.toml",
-                import
-                    .add_profile
-                    .clone()
-                    .expect("Failed to get profile name")
-            )
-        } else {
-            "--add-profile flag was not set. No profile added to snfoundry.toml".to_string()
-        },
+        add_profile: import.add_profile.as_ref().map_or_else(
+            || "--add-profile flag was not set. No profile added to snfoundry.toml".to_string(),
+            |profile_name| format!("Profile {profile_name} successfully added to snfoundry.toml"),
+        ),
+        account_name: account.map_or_else(
+            || Some(format!("Account imported with name: {account_name}")),
+            |_| None,
+        ),
```

Comment:
Due to the limitation on the acceptable lines of code for a function, I have to refactor this to save up some lines as `cargo clippy` will flag the `import` function as having too many lines (>100).